### PR TITLE
[CARBONDATA-2370] Added document for presto multinode setup for carbondata

### PR DESCRIPTION
**Pull request by geetikagupta16**
Be sure to do all of the following checklist to help us incorporate 
your contribution quickly and easily:

 - [ ] Any interfaces changed?
 
 - [ ] Any backward compatibility impacted?
 
 - [ ] Document update required?

 - [ ] Testing done
        Please provide details on 
        - Whether new unit test cases have been added or why no new tests are required?
        - How it is tested? Please attach test report.
        - Is it a performance related change? Please attach the performance test report.
        - Any additional information to help reviewers in testing this change.
       
 - [ ] For large changes, please consider breaking it into sub-tasks under an umbrella JIRA.

### CODE REVIEW

**Comment by CarbonDataQA** [comment]
Build Success with Spark 2.1.0, Please check CI http://136.243.101.176:8080/job/ApacheCarbonPRBuilder1/5238/

**Comment by CarbonDataQA** [comment]
Build Success with Spark 2.2.1, Please check CI http://88.99.58.216:8080/job/ApacheCarbonPRBuilder/4059/

**Comment by geetikagupta16** [comment]
@sgururajshetty I have made the required changes. Please review

**Comment by CarbonDataQA** [comment]
Build Success with Spark 2.2.1, Please check CI http://88.99.58.216:8080/job/ApacheCarbonPRBuilder/4140/

File: integration/presto/Presto_Cluster_Setup_For_Carbondata.md [hunk 1]
```diff
@@ -0,0 +1,133 @@
...
+## Catalog Configurations
+
+1. Create a folder named `catalog` in etc directory of presto on all the nodes of the cluster including the coordinator.
+
+##### Configuring Carbondata in Presto
+1. Create a file named `carbondata.properties` in the `catalog` folder and set the required properties on all the nodes.
+
+## Add Plugins
+
+1. Create a directory named `carbondata` in plugin directory of presto
```
Comment (sgururajshetty) [inline integration/presto/Presto_Cluster_Setup_For_Carbondata.md]: Period at the end of sentence for both the point. Check for all the sentence

File: integration/presto/Presto_Cluster_Setup_For_Carbondata.md [hunk 2]
```diff
@@ -0,0 +1,133 @@
...
+## Start Presto CLI
+```
+./presto
+```
+To connect to carbondata catalog use the following command:
+
+```
+./presto --server <coordinator_ip>:8080 --catalog carbondata --schema <schema_name>
+```
+Execute the following command to ensure the workers are connected
```
Comment (sgururajshetty) [inline integration/presto/Presto_Cluster_Setup_For_Carbondata.md]: : end of sentence

**Comment by CarbonDataQA** [comment]
Build Failed  with Spark 2.1.0, Please check CI http://136.243.101.176:8080/job/ApacheCarbonPRBuilder1/5322/

**Comment by geetikagupta16** [comment]
@sgururajshetty @chenliang613. Please review

**Comment by sgururajshetty** [comment]
LGTM

**Comment by geetikagupta16** [comment]
@chenliang613 Please review this PR

File: integration/presto/Presto_Cluster_Setup_For_Carbondata.md [hunk 3]
```diff
@@ -0,0 +1,133 @@
...
+
+ The default minimum level is `INFO`. There are four levels: `DEBUG`, `INFO`, `WARN` and `ERROR`.
+
+## Coordinator Configurations
+
+  ##### Contents of your config.properties
+  ```
+  coordinator=true
+  node-scheduler.include-coordinator=false
+  http-server.http.port=8080
```
Comment (chenliang613) [inline integration/presto/Presto_Cluster_Setup_For_Carbondata.md]: Here, the example port, don't suggest using 8080, it is easy to generate conflict.

**Comment by geetikagupta16** [comment]
@chenliang613 I have made the required changes. Please check

**Comment by CarbonDataQA** [comment]
Build Success with Spark 2.1.0, Please check CI http://136.243.101.176:8080/job/ApacheCarbonPRBuilder1/5860/

**Comment by ravipesala** [comment]
SDV Build Success , Please check CI http://144.76.159.231:8080/job/ApacheSDVTests/4903/

**Comment by CarbonDataQA** [comment]
Build Success with Spark 2.2.1, Please check CI http://88.99.58.216:8080/job/ApacheCarbonPRBuilder/4706/

**Comment by chenliang613** [comment]
LGTM

**Pull request closed without merging**